crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2712,6 +2720,7 @@ impl AuthorityPerEpochStore {
         cache_reader: &dyn ExecutionCacheRead,
         commit_round: Round,
         previously_deferred_tx_digests: HashSet<TransactionDigest>,
+        randomness_manager: &mut Option<tokio::sync::MutexGuard<'_, RandomnessManager>>,
```

Comment:
you can probably just pass a `&mut RandomnessManager` which will be cleaner, right? Doesn't tie all this code to a specific lock type

Reply:
changed to `Option<&mut RandomnessManager>`